haystack/nodes/prompt/invocation_layer/open_ai.py

```diff
@@ -35,7 +35,7 @@ def __init__(
         api_key: str,
         model_name_or_path: str = "gpt-3.5-turbo-instruct",
         max_length: Optional[int] = 100,
-        api_base: str = "https://api.openai.com/v1",
+        api_base: Optional[str] = "https://api.openai.com/v1",
```

Comment:
What's the purpose of this change? It's essentially a no-op, i.e., passing `None` is the same as just using the default value. I'd revert it.